src/LSST/M1M3/SS/FPGA/SimulatedFPGA.cpp

```diff
@@ -118,8 +151,18 @@ void SimulatedFPGA::pullTelemetry() {
     this->supportFPGAData.InclinometerErrorTimestamp = 0;
     this->supportFPGAData.InclinometerErrorCode = 0;
     this->supportFPGAData.InclinometerSampleTimestamp = timestamp;
+
+    // This comment is made in src/LSST/M1M3/SS/Inclinometer/Inclinometer.cpp
+    // 0 = Zenith, we want 0 to = Horizon
+    // Also we want below horizon to be negative
+    // Which I take to mean that the inclinometer is measuring Zenith angle,
+    // so I will convert elevation to zenith angle here.
+
+    this->elevationReadWriteLock.lock();
     this->supportFPGAData.InclinometerAngleRaw =
-            (int32_t)(tmaElevation.Elevation_Angle_Actual * 1000.0) + (this->getRnd() * 5.0);
+            (int32_t)((this->mountElevation - 90.0) * 1000.0) + (this->getRnd() * 5.0);
```

Comment:
This should be (90 - this->mountElevation) * 1000. By trivial check, elevation angle 45 deg is zenith angle 45 deg. Your code will show -45 (which technicaly is the same, but I do prefer positive values for zenith angle measurements)

Reply:
yes, but check the math done here: https://github.com/lsst-ts/ts_m1m3support/blob/c8c90ba18b2351963a69ff810f2834dbfd6cac19/src/LSST/M1M3/SS/Inclinometer/Inclinometer.cpp#L93

I think it is wrong there instead. Should I correct this? I am afraid to change it there as this was tested with the actual hardware and I am not sure if it is used or not. If I correct the math here I have to correct it there as well...

Reply:
Just to be clear. If I do 

```
InclinometerAngleRaw = (90 - mountElevation) * 1000.
```

which is the correct, then,

```
angle = 90.0 - InclinometerAngleRaw / 1000.0
```

which is not what `inclinometer.cpp` is computing... So either I correct both places or leave it the way it is in both places.. But `inclinometer.cpp` was tested with real hardware... so I really don't know if this is how it is supposed to be or a previous mistake.